consensus/src/processes/coinbase.rs

```diff
@@ -389,7 +359,7 @@ mod tests {
                         29, 146, 203, 117, 108, 2, 197, 96, 172,
                     ],
                 ),
-                extra_data: &[48, 46, 49, 50, 46, 56, 47],
+                extra_data: &[48u8, 46, 49, 50, 46, 56, 47] as &[u8],
```

Comment:
Why 48u8 is needed?

Reply:
Extra data needs to be cast to a slice `&[u8]` explicitly because `extra_data` field is generic (and we compare the test result to such a slice). So this array needs explicit typing before the cast.